railties/lib/rails/generators/base.rb

```diff
@@ -252,6 +252,7 @@ def self.inherited(base) #:nodoc:
         def class_collisions(*class_names)
           return unless behavior == :invoke
           return if options.skip_collision_check?
+          return if options.force?
```

Comment:
I'm not sure if we want to define `ignore_collisions?` and chain these two together. I thought it was better left as is to indicate that this is explicitly skipped from options.